src/cdk/tree/nested-node.ts

```diff
@@ -60,6 +60,7 @@ export class CdkNestedTreeNode<T> extends CdkTreeNode<T> implements AfterContent
               protected _tree: CdkTree<T>,
               protected _differs: IterableDiffers) {
     super(_elementRef, _tree);
+    this._elementRef.nativeElement.classList.add('cdk-tree-node', 'cdk-nested-tree-node');
```

Comment:
Can you add a comment here that explains why you add the classes directly instead of using the `host` property?

(similar below)